sdk/core/azure-core/build.gradle

```diff
@@ -1,6 +1,6 @@
 ext.publishName = "Microsoft Azure Android Core Library"
 description = "This package contains core types for Azure Android clients."
-version = "1.0.0-beta.1"
+version = "1.0.0-beta.2"
```

Comment:
Is this the only place we should replace the version number?

Reply:
Yes, this is the only place.